planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -990,13 +996,17 @@ SlowDownSection ObstacleStopPlannerNode::createSlowDownSection(
   const double dist_remain, const double dist_baselink_to_obstacle,
   const VehicleInfo & vehicle_info, const double current_acc)
 {
+  mutex_.lock();
   if (!current_velocity_ptr_) {
     // TODO(Satoshi Ota)
+    mutex_.unlock();
     return SlowDownSection{};
   }
 
+  const auto current_vel = current_velocity_ptr_->twist.twist.linear.x;
+  mutex_.unlock();
+
```

Comment:
```suggestion
  const VehicleInfo & vehicle_info, const double current_acc, const double current_vel)
{
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1330/commits/a472dd9417fbf6fc1b17998501f87a44e393ebdb